src/quo2/components/navigation/page_nav/view.cljs

```diff
@@ -28,6 +30,7 @@
              :icon-only?          true
              :size                32
              :on-press            on-press
+             :background          (when (button-properties/backgrounds background) background)
```

Comment:
Just be careful with this approach.

Sometimes, I have noticed that when giving `nil` as color, the component doesn't use `:transparent`, it uses the color previously given and doesn't remove it. 

If in this case it's working properly, then keep the code as is 👍

Reply:
I agree @ulisesmac. If the component needs to have a fallback (like `:transparent`), This approach is not correct.

But for the `background` prop, as far as I saw, the components such as `button`, and `dropdown` have only two values `blur` / `photo`. There is no fallback for this property value and the component accepts `nil` values.